packages/ffe-collapse-react/src/Collapse.js

```diff
@@ -1,82 +1,43 @@
-import React, { useRef, useState, useEffect } from 'react';
-import { string, bool, func, object } from 'prop-types';
+import React, { useRef } from 'react';
+import { string, bool, func, node } from 'prop-types';
 import classNames from 'classnames';
 
-const Collapse = ({ className, style, isOpen, onRest, ...rest }) => {
-    const content = useRef();
-    const [height, setHeight] = useState(() => (isOpen ? 'auto' : '0'));
-    const [overflow, setOverflow] = useState(() =>
-        isOpen ? 'visible' : 'hidden',
-    );
-    const [visibility, setVisibility] = useState(() =>
```

Comment:
Burde kanskje beholde visibility hidden når den er lukket? Da kan ikke input-felt fokuseres når den er lukket. Ser det ikke er et problem for accordion for der rendres ikke innholdet hvis lukket

Reply:
Det er ett veldig gott poeng. Det skall jag få fikset

Reply:
Har opdatert https://black-beach-0d62d0d03-1872.westeurope.2.azurestaticapps.net/